news/_posts/releases/2022-08-08-prestashop-8-0-beta-release.md

```diff
@@ -0,0 +1,121 @@
+* [Translations packs are not up-to-date for 8.0](https://github.com/PrestaShop/PrestaShop/issues/29017)
+* [Multistore - A new product created in all shops is not displayed on the products list](https://github.com/PrestaShop/PrestaShop/issues/27599)
+* [Contextual help in the Back office is not fully functional](https://github.com/PrestaShop/PrestaShop/issues/29169)
+* [Errors are displayed when upgrading from 1.7.8.6 to 8.0.0](https://github.com/PrestaShop/PrestaShop/issues/29248)
+* Some modules are not yet updated to work with 8.0 and might break your shop. We recommend disabling the following modules before upgrading to this beta:
+  * Onboarding (welcome)
+  * PrestaShop Metrics (ps_metrics)
+  * PrestaShop Facebook (ps_facebook)
+  * Prestashop Checkout (ps_checkout)
+  * Google marketing (psxmarketingwithgoogle)
+  * Marketplace in your Back office (ps_mbo)
+
+The [complete list of known bugs](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aopen+is%3Aissue+label%3A8.0.x+label%3ABug) is constantly updated on GitHub.
+
+
+## Acknowledgements
+
+A big thank you to the **113 people** who contributed **more than 1000 Pull Requests** included in this version: Adib Aroui, Alexandre Bru, Amaury Hanser, Amazzing, Amit Kumar Tiwari, Andromaque, Arnaud104, Ashley Butcher, Axome, Beattie282, Bertrand Merlier, Boubker Bribri, Brais Pato, Bruno Desprez, Camille V, Christophe Rouille, Clotaire Renaud, Constantin Boulanger, Damien Tupinier, Daniel Hlavacek, David Diverres, David Gonzalez, David Oury, Dominik Ulrich, e-gaulue, Fabien Papet, Franck Lefèvre, gennaris, Georges Cubas, Henri Davigne, Hervé Hennes, Hibatallah Aouadni, Hugues Gobet, Ibrahima Sow, idnovate, Igor Stępień, Jean-François Viguier, Jérôme Wohlschlegel, Jevgenij Visockij, Jonathan Danse, Jonathan Lelievre, Jory Hogeveen, Julie Varisellaz, Julius Žukauskas, Karlis Suvi, Kévin Gleizes, Khouloud Belguith, Krystian Podemski, Kyary, Lionel Massin, Lorenz Meyer, Louis Authie, Luc Vigato, Mahmoud Parvazi, Marco Ingraiti, Marco Salvatore, Marcos, Marek Jedrzejewski, Marion François, Marius Gudauskis, Marwa Chelly, Mathieu Ferment, Matthias Raigne, Matthieu Rolland, Michael Voříšek, Mickaël Fernandez, Moncef Essid, mrAKAR, Nesrine Abdmouleh, Nicolas Sorosac, NoZTurn, okom3pom, Olivier Le Corre, Pablo Borowicz, Pascal Cescon, Pedro Nucci, Peter Schaeffer, Philip Iezzi, Pierre Rambaud, Pierre-Yves Christmann, PrestaAlba, prestaforum, Prestaplugins, Prestaworks, Priyank Desai, Psyray, PululuK, Ricardo González Garza, Robin Fischer, Romain ‘Maz’ BILLOIR, Romain Couderc, Sam, Sauli Maijala, Sébastien Alfaiate, Sergio Quiñonez, Seth Vargo, Sharak, Simone, Simone Gennari, Tatu Wikman, Thomas Baccelli, Thomas Leguillant, Tom Combet, Tuni-Soft, Valentin Szczupak, venditdevs, Vincent Garcia, Viorel Preoteasa, Vladimir, Web Helpers Pau, Yneet, yo7hli, and Yohann Tilotti.
```

Comment:
```suggestion
A big thank you to the **113 people** who contributed **more than 1000 Pull Requests** included in this version: Adib Aroui, Alexandre Bru, Amaury Hanser, Amazzing, Amit Kumar Tiwari, Andromaque, Arnaud104, Ashley Butcher, Axome, Beattie282, Bertrand Merlier, Boubker Bribri, Brais Pato, Bruno Desprez, Camille V, Christophe Rouille, Clotaire Renaud, Constantin Boulanger, Damien Tupinier, Daniel Hlavacek, David Diverres, David Gonzalez, David Oury, Dominik Ulrich, e-gaulue, Fabien Papet, Franck Lefèvre, gennaris, Georges Cubas, Henri Davigne, Hervé Hennes, Hibatallah Aouadni, Hugues Gobet, Ibrahima Sow, idnovate, Igor Stępień, Jean-François Viguier, Jérôme Wohlschlegel, Jevgenij Visockij, Jonathan Danse, Jonathan Lelievre, Jory Hogeveen, Julie Varisellaz, Julius Žukauskas, Karlis Suvi, Kévin Gleizes, Khouloud Belguith, Krystian Podemski, Kyary, Lionel Massin, Lorenz Meyer, Louis Authie, Luc Vigato, Mahmoud Parvazi, Marco Ingraiti, Marco Salvatore, Marcos, Marek Jedrzejewski, Marion François, Marius Gudauskis, Marwa Chelly, Mathieu Ferment, Matthias Raigne, Matthieu Rolland, Michael Voříšek, Mickaël Fernandez, Moncef Essid, mrAKAR, Nesrine Abdmouleh, Nicolas Sorosac, NoZTurn, Olivier Le Corre, Pablo Borowicz, Pascal Cescon, Pedro Nucci, Peter Schaeffer, Philip Iezzi, Pierre Rambaud, Pierre-Yves Christmann, PrestaAlba, prestaforum, Prestaplugins, Prestaworks, Priyank Desai, Psyray, PululuK, Ricardo González Garza, Robin Fischer, Romain ‘Maz’ BILLOIR, Romain Couderc, Sam Berry, Sauli Maijala, Sébastien Alfaiate, Sergio Quiñonez, Seth Vargo, Sharak, Simone, Simone Gennari, Tatu Wikman, Thomas Baccelli, Thomas Leguillant, Tom Combet, Tuni-Soft, Thomas Roux, Valentin Szczupak, venditdevs, Vincent Garcia, Viorel Preoteasa, Vladimir, Web Helpers Pau, Yneet, yo7hli, and Yohann Tilotti.
```

Reply:
uhh, the diff is broken, I've changed `Sam` to `Sam Berry` and `okom3pom` to `Thomas Roux`

Reply:
Okom3pom = Thomas Roux :+1: